packages/IMAPClient-Core.package/ICFileReadWriter.class/instance/endpoint..st

```diff
@@ -1,3 +1,3 @@
-private
+accessing
```

Comment:
The intention of the private category was to hide these methods from other classes. I do not think it is better to put them back into accessing.

Reply:
As a discussion showed that the border between "intended to be private" and "just not used by other classes jet" is unclear in most cases, we are not going to refactor all usages of private variables (yet). In order to keep the code consistent nevertheless we are not using a private category in here either.